WIP: PR for new last-minute talks.

FlipperPA opened the pull request:
## Description

Work in progress: last-minute talk additions.

Still needs:

* Finish up text of new talk MD files - DONE!
* Add speaker bios - DONE!
* Move old talk MD files to appropriate new spots  - 1/2 done?
* Social banners for new talks - DONE!

tim-schilling commented on _presenters/tim-allen.md:
```diff
@@ -0,0 +1,24 @@
+long before Bitcoin, and enjoys beating his head against brick walls (as demonstrated
+by his passion for coding, cycling in Philadelphia, and never-ending quest to get his
+cat to behave). A life-long Philadelphia resident, he graduated from the University
+of Pennsylvania in 1996 with an individualized major, “Computer Aided Information Acquisition”.
```
@FlipperPA I grabbed your bio from last year. Let me know if we need any changes.

FlipperPA replied:
Thank You!

tim-schilling commented on static/img/social/presenters/dawn-wages.png:
To anyone wondering why this changed, it's because Dawn has multiple talks (the tutorial) and it was rendering oddly. I temporarily removed her slug from the tutorial to regenerate this.

drewbrew replied:
You can also override her talk title from the speaker details. See Paul or Calvin for an example

tim-schilling commented:
I think the remaining todo is "Move old talk MD files to appropriate new spots".

drewbrew reviewed (commented):
Without loading it locally, I think this is right so far

FlipperPA commented:
I've moved the two talks to their new slots and changed the file name to `t2` for online. I think I got it right, but would very much like a second and third set of eyes. If it looks good to both of you @drewbrew and @tim-schilling, please feel free to merge and publish. I'll be coaching DjangoGirls and working on slides tomorrow!
